src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -106,7 +109,7 @@ var createFiber = function(tag : number, key : null | string) : Fiber {
     memoizedInput: null,
     output: null,
 
-    hasPendingChanges: true,
+    pendingWorkPriority: 0,
```

Comment:
I mean I'll update this to use `NoWork` for clarity. Just skeptical about going too far on the opaqueness / abstractions here.